.github/workflows/publish.yml

```diff
@@ -114,59 +130,29 @@ jobs:
           TENCENT_COS_SECRETID: ${{ secrets.TENCENT_COS_SECRETID }}
           TENCENT_COS_SECRETKEY: ${{ secrets.TENCENT_COS_SECRETKEY }}
         run: |
-          sudo -E ./utils/publish-rpm.sh init_cos_utils centos
-
-      - name: RPM repo init for centos
-        run: |
-          sudo -E ./utils/publish-rpm.sh repo_init centos
-          sudo -E ./utils/publish-rpm.sh repo_clone centos
+          sudo -E ./utils/publish-rpm.sh init_cos_utils
 
-      - name: RPM repo init for redhat ubi
+      - name: RPM repo init for centos and redhat
         run: |
-          sudo -E ./utils/publish-rpm.sh repo_init redhat
-          sudo -E ./utils/publish-rpm.sh repo_clone redhat
+          sudo -E ./utils/publish-rpm.sh repo_init
+          sudo -E ./utils/publish-rpm.sh repo_clone
 
-      - name: RPM repo package update
+      - name: RPM repo package update 
         env:
           GPG_NAME: ${{ secrets.GPG_NAME }}
           GPG_MAIL: ${{ secrets.GPG_MAIL }}
         run: |
           echo "${{ secrets.RPM_GPG_PRIV_KEY }}" >> /tmp/rpm-gpg-publish.private
           echo "${{ secrets.RPM_GPG_PASSPHRASE }}" >> /tmp/rpm-gpg-publish.passphrase
           echo "${{ secrets.RPM_GPG_PUB_KEY }}" >> /tmp/rpm-gpg-publish.public
-          sudo -E ./utils/publish-rpm.sh rpm_gpg_sign centos
-          sudo -E ./utils/publish-rpm.sh repo_package_sync centos
-
-      - name: RPM repo backup for centos
```

Comment:
@Revolyssup keep this code I think if not POC.

Reply:
Re-added it